problems/[230601] K Closest Points to Origin.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * leetcode problem link: https://leetcode.com/problems/k-closest-points-to-origin
+ *
+ * @param {number[][]} points
+ * @param {number} k
+ * @return {number[][]}
+ */
+const kClosest = function (points, k) {
+  const distance = [];
+
+  for (let i = 0; i < points.length; i += 1) {
+    distance.push(Math.sqrt(points[i][0] ** 2 + points[i][1] ** 2));
+  }
```

Comment:
```suggestion
  const distance = points.map((point) => {
    return Math.sqrt(point[0] ** 2 + point[1] ** 2)
  });
```

distance 변수는 for문에서 추가되고 관리되는데, 동일한 역할을 map() 메소드에서 하면 더 이해하기 쉬워지고 안전하게 관리할 수 있을 것 같습니다~

Reply:
좋은 답변 감사합니다!